src/tools/generate/generate_chapters.js

```diff
@@ -3,6 +3,9 @@ const showdown = require('showdown');
 const ejs = require('ejs');
 const prettier = require('prettier');
 
+//Chapters may exist but not be ready to be launched so do not include in sitemap
```

Comment:
In validate.py, `SUPPORTED_YEARS` maps years to supported languages. We should import that to avoid two sources of truth.

Reply:
Agreed, but didn't know how to do that since this is just JS and not Python.

Any pointers or similar variable that is being pulled in from Python to JS?

Reply:
Oh true. Hmmm... this isn't a launch blocker so let's keep as-is and merge.